src/collider.cxx

```diff
@@ -108,7 +108,7 @@ std::tuple<double, int> Collider::sample_collision() {
 
   do {
     // Sample b from P(b)db = 2*pi*b.
-    b = bmin_ + (bmax_ - bmin_) * std::sqrt(random::canonical<double>());
+    b = std::sqrt(bmin_ * bmin_ + (bmax_ * bmax_ - bmin_ * bmin_) * random::canonical<double>());
```

Comment:
I think I understand this, but just to confirm, the point here is to sample b^2 uniformly?

Reply:
Yes. Actually, the patch is not mine. I took it from `UrQMD` when the chosen option for impact parameter weighting `CTO5` is set to `1`, `random b from bmin to bmax, bdb weighted` which we used to use when generating events: `init.f L.421`